common/autoconf/toolchain.m4

```diff
@@ -360,6 +360,10 @@ AC_DEFUN_ONCE([TOOLCHAIN_POST_DETECTION],
   CFLAGS="$ORG_CFLAGS"
   CXXFLAGS="$ORG_CXXFLAGS"
   OBJCFLAGS="$ORG_OBJCFLAGS"
+
+  # filter out some unwanted additions autoconf may add to CXX; we saw this on macOS with autoconf 2.72
```

Comment:
Pls update the Oracle copyright date, the original fix modified the end date to 2024. basics.m4 may also need an update.

Reply:
Both files have contributions copied from a 2024 change to jdk11u: Both now have a 2024 end date.